scripts/setup_letsencrypt.sh

```diff
@@ -11,8 +11,8 @@ git config --global user.email "cpcsai.nimbus@gmail.com"
 git config --global user.name "GCE_GIT_BOT"
 cd /api-certificates
 tar xvf letsencrypt_backup.tar -C /
-crontab /nimbus/letsencrypt-backup-tar.cron
+crontab /nimbus/scripts/letsencrypt-backup-tar.cron
```

Comment:
gets the cron job running

https://explainshell.com/explain?cmd=crontab+%2Fnimbus%2Fscripts%2Fletsencrypt-backup-tar.cron